README.md

```diff
@@ -6,6 +6,20 @@ If you are looking to explore the Dapr documentation, please go to the documenta
 
 This repo contains the markdown files which generate the above website. See below for guidance on running with a local environment to contribute to the docs.
 
+## Branch guidance
+
+The Dapr docs handles branching differently than most code repositories. Instead of having a `master` or `main` branch, every branch is labeled to match the major and minor version of a runtime release.
+
+The following branches are currently maintained:
+
+| Branch | Website | Description |
+|--------|---------|-------------|
+| [v0.11](https://github.com/dapr/docs) (primary) | https://docs.dapr.io | Latest Dapr release documentation. Typo fixes, clarifications, and most documentation goes here.
+| [v1.0-rc2](https://github.com/dapr/docs/tree/v1.0-rc2) | https://v1-rc2.docs.dapr.io/ | Latest Dapr release candidate release documentation. Doc updates that are only applicable to v1.0-rc2+ go here.
```

Comment:
In v1.0-rc2+ lose the + here?

Reply:
Depends on if we want to merge down or up to spread the change to other branches. Since new features are being written in the top-most branch it will probably be easier to merge from rc2 into rc3, so changes that are applicable to both would go into rc2